examples/ido_pool/tests/ido_pool.js

```diff
@@ -0,0 +1,430 @@
+const anchor = require('@project-serum/anchor');
```

Comment:
Let's add this to travis in the second examples section.

Reply:
Is this something I need to do?

Reply:
Done.